src/pudl/metadata/resources/ferc1.py

```diff
@@ -437,6 +437,29 @@
         "etl_group": "ferc1",
         "field_namespace": "ferc1",
     },
+    "depreciation_amortization_summary_ferc1": {
+        "description": "Summary of depreciation, and amortization of electric plant.",
```

Comment:
We should make sure all of the table descriptions indicate what FERC 1 schedule (page number) the data came from. In this case it's Schedule 336a.

Reply:
Added